docker-compose.yaml

```diff
@@ -91,8 +91,8 @@ services:
 
   frontend:
     build:
-      context: ./frontend
-      dockerfile: docker/dev/Dockerfile
+      context: .
```

Comment:
5. The context of the dockerfile needs to be updated because we need to read the config.json file